lang/en.json

```diff
@@ -396,5 +410,8 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Delete Item",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "Are you sure you want to delete this item?",
   "YZECORIOLIS.AutomaticFire": "Automatic fire",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Number of ignored 1s (max 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Number of ignored 1s (max 2)",
+
+  "YZECORIOLIS.RollTotal": "Total rolled",
+  "YZECORIOLIS.RollSubstitute": "2d6-saving throw instead of"
```

Comment:
I think the wording elsewhere in the code is a `desperation roll` so I'd suggest something like `desperation roll (2d6)` maybe?

Reply:
yes, I will change that :)